src/mixer/plugins/volume_mapping.cxx

```diff
@@ -31,8 +31,6 @@
  */
 
 #define _ISOC99_SOURCE /* lrint() */
-#define _GNU_SOURCE /* exp10() */
```

Comment:
I have only used plain git (and also managed to get it to work here, but I will have a look on stgit, that I didn't know about. Thanks!